governance/program/src/instruction.rs

```diff
@@ -234,6 +238,25 @@ pub enum GovernanceInstruction {
         signatory: Pubkey,
     },
 
+    /// Adds a required signatory to the Governance, which will be applied to all proposals created with it
+    ///
+    ///   0. `[writable, signer]` The Governance account the config is for
+    ///   1. `[writable]` RequiredSignatory Account
+    ///   2. `[signer]` Payer
+    ///   3. `[]` System program
+    AddRequiredSignatory {
+        #[allow(dead_code)]
+        /// Required signatory to add to the Governance
+        signatory: Pubkey,
+    },
+
+    /// Removes a required signatory from the Governance
+    ///
+    ///  0. `[writable, signer]` The Governance account the config is for
+    ///  1. `[writable]` RequiredSignatory Account
+    ///  2. `[writable]` Beneficiary Account which would receive lamports from the disposed RequiredSignatory Account
+    RemoveRequiredSignatory,
```

Comment:
`RemoveRequiredSignatory ` must be added at the end of the struct for backward compatibility of the instructions ids

Reply:
resolved